src/transformer-components/transformerList.ts

```diff
@@ -339,32 +339,27 @@ const transformerList: TransformerList = {
         context1: {
           title: "First Dataset to Compare",
```

Comment:
This should probably just be "Dataset to Compare" or even "Dataset to Compare Within"